qfieldsync/utils/exceptions.py

```diff
@@ -19,17 +19,12 @@
  ***************************************************************************/
 """
 
-# coding=utf-8
-from qgis.core import (
-        QgsMessageLog,
-        Qgis
-)
+from qgis.core import QgsMessageLog, Qgis
```

Comment:
The idea of the multiline import is to keep them trackable in git

Reply:
the IDE makes it clear which is used or not.
is there another purpose?

Reply:
It's about git history, not the ide

Reply:
I just don't see the interest in tracking import changes and I prefer the cleaner code layout.
But I don't mind reverting, just tell me

Reply:
I'd like to have a common style here, I was always switching things TO this style, but switching forth and back seems counterproductive.

We could also go for

```
from x import Y
from x import Z
```

that would also avoid extremely long lines